.github/workflows/publish-command.yml

```diff
@@ -130,6 +130,8 @@ jobs:
           MIXPANEL_INTEGRATION_TEST_CREDS: ${{ secrets.MIXPANEL_INTEGRATION_TEST_CREDS }}
           MSSQL_RDS_TEST_CREDS: ${{ secrets.MSSQL_RDS_TEST_CREDS }}
           PAYPAL_TRANSACTION_CREDS: ${{ secrets.SOURCE_PAYPAL_TRANSACTION_CREDS }}
+          ORACLE_SSH_KEY_TEST_CREDS: ${{ secrets.ORACLE_SSH_KEY_TEST_CREDS }}
```

Comment:
can we instead use the approach you took in #5811 ?

Reply:
sure, i'll modify on container approach once https://github.com/airbytehq/airbyte/pull/6312/ will be merged

Reply:
@sherifnada I changed the integration tests for the bastion container approach

Reply:
@sherifnada also how do you think maybe we need to create separate tasks to also change the integration tests for Oracle Source, MySQL Source and Destination, MSSQL Source and Destination with docker bastion container?

Reply:
@VitaliiMaltsev yup let's create issues on the backlog to do this, and make sure with the team that any new SSH implementations follow the bastion approach 

cc @alexandr-shegeda